android/src/main/kotlin/io/didomi/fluttersdk/EntitiesHelper.kt

```diff
@@ -41,5 +42,13 @@ class EntitiesHelper {
             val json = gson.toJsonTree(valid)
             return gson.fromJson(json, HashMap::class.java)
         }
+
+        // Convert a user status into a map.
+        @Throws(JsonParseException::class)
```

Comment:
now that I see that we have 3 toMap methods, do you think that it would make sense to have just 1 that can  be reused?

Reply:
Done